ranger/data/scope.sh

```diff
@@ -290,6 +292,45 @@ handle_mime() {
             xls2csv -- "${FILE_PATH}" && exit 5
             exit 1;;
 
+        ## SQLite
+        *sqlite3)
+            ## Preview as text conversion
+            local tables=""
+            [ "$SQLITE_TABLE_LIMIT" -ge 0 ] || SQLITE_TABLE_LIMIT=20  # fallback to 20 if an invalid value is given
+            [ "$SQLITE_ROW_LIMIT" -ge 0 ] || SQLITE_ROW_LIMIT=5       # fallback to 5 if an invalid value is given
+            if tables="$( sqlite3 "file:${FILE_PATH}?mode=ro" \
+                          "SELECT name FROM sqlite_master \
+                           WHERE type='table' AND name NOT LIKE 'sqlite_%' LIMIT ${SQLITE_TABLE_LIMIT};" )"; then
+                if [[ -n "$(sqlite3 "file:${FILE_PATH}?mode=ro" '.tables')" ]]; then
```

Comment:
Don't use `[[ ]]` that's a bashism.
And avoid nesting assignment in if statement conditions for clarity.

Reply:
Refactored.